pages/admin/finance/index.tsx

```diff
@@ -17,29 +46,36 @@ export default function financePage() {
     }
 
     if (estimatedCtx !== null) {
+      if (Chart.getChart(estimatedCtx)) {
+        Chart.getChart(estimatedCtx)?.destroy();
+      }
       const chart = new Chart(estimatedCtx, {
         type: 'pie',
         data: {
-          labels: ['Research Funding', 'Dev funding', 'Food', 'Merchandise', 'Other'],
+          labels: divisionNames,
           datasets: [
             {
               label: '% of Budget',
```

Comment:
I believe this should be something like `Amount used` or anything that signify how much they really spent instead of percentage.